File: client/src/app/modules/history/models/activity.ts
export interface Activity {
    id: number;
    cardName: string;
    activityName: string;
    listName: string;
    previous: string;
    updated: string;
    date: Date;
}

export interface ActivityResponse {
    activities: Activity[];
    pageSize: number;
    totalCount: number;
}

File: API/Entities/Activity.cs
namespace API.Entities
{
    public class Activity
    {
        public int Id { get; set; }
        public string CardName { get; set; }
        public int CardId { get; set; }
        public int BoardId { get; set; }
        public string ActivityName { get; set; }
        public string ListName { get; set; }
        public string Previous { get; set; }
        public string Updated { get; set; }
        public DateTime Date { get; set; }
    }
}